packages/admin-api-client/src/graphql/client.ts

```diff
@@ -78,7 +75,7 @@ export function createAdminApiClient({
   };
 
   const graphqlClient = createGraphQLClient({
-    headers: config.headers,
+    headers: config.headers ?? {},
```

Comment:
Same as SFAPI's change - this can likely go away if we remove the change in the graphql-client util types